pkgs/by-name/vd/vdhcoapp/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, fetchFromGitHub
+, buildNpmPackage
+, toml2json
+, nodejs
+, ffmpeg
+, callPackage
+, filepicker ? callPackage ./filepicker.nix {}
+, substituteAll
+, makeWrapper
+}:
+
+# This is an adaptation with buildNpmPackage based on https://github.com/milahu/nur-packages/commit/3022ffb3619182ffcd579194e1202e3978e4d55b
+buildNpmPackage rec {
+  pname = "vdhcoapp";
+  version = "2.0.19";
+  srchash = "sha256-8xeZvqpRq71aShVogiwlVD3gQoPGseNOmz5E3KbsZxU=";
+
+  src = fetchFromGitHub {
+    owner = "aclap-dev";
+    repo = "vdhcoapp";
+    rev = "v${version}";
+    hash = srchash;
+  };
+
+  sourceRoot = "${src.name}/app";
+  npmDepsHash = "sha256-E032U2XZdyTER6ROkBosOTn7bweDXHl8voC3BQEz8Wg=";
+  dontNpmBuild = true;
+
+  nativeBuildInputs = [
+    toml2json
+    makeWrapper
+  ];
+
+  patches = [
+    (substituteAll {
+      src = ./ffmpeg-filepicker.patch;
+      inherit ffmpeg;
+      filepicker = lib.getExe filepicker;
+    })
+  ];
+
+  postPatch = ''
+    # Cannot use patch, setting placeholder here
+    substituteInPlace src/native-autoinstall.js \
+      --replace process.execPath "\"${placeholder "out"}/bin/vdhcoapp\""
+  '';
+
+  preBuild = ''
+    toml2json --pretty ../config.toml > src/config.json
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/opt/vdhcoapp
+
+    cp -r . "$out/opt/vdhcoapp"
+
+    makeWrapper ${nodejs}/bin/node $out/bin/vdhcoapp \
+      --add-flags $out/opt/vdhcoapp/src/main.js
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Companion application for the Video DownloadHelper browser add-on";
+    homepage = "https://www.downloadhelper.net/";
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ wolfangaukang hannesgith ];
```

Comment:
If i understood @wolfangaukang correctly he is not willing to maintain that package in nixpkgs, so i assume that listing here as maintainer is a false claim, isnt it?

Reply:
you are right, my thought was that this code was mainly his contribution and he added himself as maintainer in the code he provided i did not deem it fair to remove him, but as he explicity wished to not maintain this package you are totally right in removing him from the maintainers list

Reply:
Yeah fair claim, maybe he responds and gives context (if he wants to be added).
As the source is his personal repo, I assume that he adds himself as maintainer for own packages (which make sense when using some evals that depend on meta.maintainers, or just wanting to fill it completely). I am doing similar on my personal flake.